ci/Makefile

```diff
@@ -1,11 +1,36 @@
-.PHONY: run restart-proxy restart-batch-proxy restart-all-proxies
+.PHONY: check-vars run restart-proxy restart-batch-proxy restart-all-proxies
 .PHONY: setup-conda-env build-hail-ci push-hail-ci test-locally
 .PHONY: run-service deploy
 
-PROJECT = $(shell gcloud config get-value project)
+PROJECT := $(shell gcloud config get-value project)
+
+# Grabs details of first instance matching HAIL_CI_INSTANCE
+DETAILS := $(shell gcloud compute instances list --format='table(NAME,STATUS,tags.list(),EXTERNAL_IP,ZONE)' | tail -n +2 | grep "$(HAIL_CI_INSTANCE)" --color=never | grep RUNNING | head -n 1 | tr -s ' ' '\n' )
```

Comment:
This feels a bit brittle. Let's use the `--filter` operation from `gcloud`:
```
EXTERNAL_IP_AND_ZONE := $(shell gcloud compute instances list \
    --format='table(EXTERNAL_IP,ZONE)' \
    --filter='status=RUNNING AND name=$(HAIL_CI_INSTANCE) \
  | tail -n +2)
INSTANCE := $(HAIL_CI_INSTANCE)
EXTERNAL_IP := $(word 1, $(EXTERNAL_IP_AND_ZONE))
ZONE := $(word 2, $(EXTERNAL_IP_AND_ZONE))
```

Reply:
Also, we should maybe just use `HAIL_CI_INSTANCE` everywhere in place of `INSTANCE`.

Reply:
Got it. Could you elaborate on why the syntax seems brittle? I was trying to reduce dependence on manufacturer specific syntax, but that may have downsides I’m unaware of. Thanks!